voter-dapp/src/App.js

```diff
@@ -1,24 +1,47 @@
 import React from "react";
+import { MuiThemeProvider, createMuiTheme } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
 import { drizzleReactHooks } from "drizzle-react";
 import "./App.css";
+import Header from "./Header.js";
 import ActiveRequests from "./ActiveRequests.js";
-import ResolvedRequests from "./ResolvedRequests.js";
+import { red, grey } from "@material-ui/core/colors";
+import AppBar from "@material-ui/core/AppBar";
 
 function App() {
+  const theme = createMuiTheme({
+    palette: {
+      primary: red, //TODO --> find actual UMA red
```

Comment:
UMA red is `#ff4a4a`.

See the press kit for all UMA colors and styles: https://github.com/UMAprotocol/website/blob/master/press-kit/UMA_Style_Guide_v2.pdf

You should be able to plug in any hex color by just setting the property to it:
```
primary: "#ff4a4a"
```